src/shared/telemetry/defaultTelemetryService.ts

```diff
@@ -109,7 +135,8 @@ export class DefaultTelemetryService implements TelemetryService {
                 await this.publisher.flush()
                 this._eventQueue.length = 0
```

Comment:
Should these be replaced with calls to `clearRecords()`?

Reply:
yes, good catch.